packages/common/core-interfaces/api-report/core-interfaces.beta.api.md

```diff
@@ -310,7 +310,6 @@ export interface ITelemetryBaseProperties {
 // @public @sealed
 export interface Listenable<TListeners extends object> {
     off<K extends keyof Listeners<TListeners>>(eventName: K, listener: TListeners[K]): void;
-    // (undocumented)
```

Comment:
Note for reviewers: this was incorrectly marked as `undocumented` seemingly due to the position of an eslint-enable comment. This method is documented.